notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb

```diff
@@ -104,7 +104,7 @@
       "source": [
```

Comment:
<ol><li>pipline -&gt; pipeline</li><li>Can you move these to the Cleanup section?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
done